cypress/fixtures/accounts.json

```diff
@@ -0,0 +1,14 @@
+{
+   "user1" : {
```

Comment:
should be a config coming from the env, we don't want this to be available inside the repository.
()